apps/storybook-react/package.json

```diff
@@ -7,40 +7,41 @@
   "scripts": {
     "build-storybook": "storybook build",
     "storybook": "storybook dev -p 6006",
```

Comment:
Upgrading dependencies and scripts according to v9 docs https://storybook.js.org/docs/migration-guide